flows/codelingo/docs/docs/parse/parse.go

```diff
@@ -0,0 +1,142 @@
+package parse
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/codelingo/clql/dotlingo"
```

Comment:
Private dependency - may not be a biggy, but means people can't build from source.